src/client/utils/dom.js

```diff
@@ -334,17 +347,12 @@ export function isDomElement (el) {
 }
 
 export function isElementInDocument (el, currentDocument) {
-    var doc        = currentDocument || document;
-    var curElement = el.parentNode;
+    var doc = currentDocument || document;
 
-    while (curElement) {
-        if (curElement === doc)
-            return true;
+    if (doc.contains)
```

Comment:
We cannot use `document.body.contains`.
Inside functions `appendChild`, `insertBefore` we can use this function for `head` element.
see https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/element.js#L201